commit e70b212eef3d0ed263f72e39026b17e726738cb7 (refs/changes/99/399/4)
Author: David Pacheco <dap@joyent.com>
Date:   2016-09-02T09:54:08-07:00 (3 years, 1 month ago)
    
    this is a test change
    Reviewed by: David Pacheco <dap@joyent.com>

diff --git a/README.md b/README.md
index f6da383..75cf6c7 100644
--- a/README.md
+++ b/README.md
@@ -1,4 +1,3 @@
 # gerrit-test
 
 This is a dummy repository for verifying cr.joyent.us behavior.
-Test update.
